support-workers/src/test/scala/com/gu/support/workers/lambdas/FailureHandlerSpec.scala

```diff
@@ -22,7 +22,7 @@ import org.scalatest.matchers.should.Matchers
 import scala.concurrent.{ExecutionContext, Future}
 
 @IntegrationTest
-class FailureHandlerIT extends AsyncLambdaSpec with MockContext {
+class FailureHandlerITSpec extends AsyncLambdaSpec with MockContext {
```

Comment:
the IT test autoimated runner only finds tests that end in "Spec" which is usually but not always the case